guide/english/vim/useful-commands/index.md

```diff
@@ -119,6 +119,9 @@ Vim has very tight integration with the `make` build automation tool. If your pr
 
 Important: if you want to replace the first occurence of the word, remove the `g` at the end.
 
+## The ".vimrc" file
+The *.vimrc* file is used to save your custom configurations so that you don't have to reconfigure your editor everytime you run vim.  To open/create your vimrc file just open open your cli and type in "vim ~/.vimrc" then add the commands you would normaly type into your editor.
```

Comment:
```suggestion
The `.vimrc` file is used to save your custom configurations so that you don't have to reconfigure your editor every time you run vim.  To open/create your `vimrc`, file just open your cli and type in `vim ~/.vimrc` then add the commands you would normally type into your editor.
```